package.json

```diff
@@ -58,7 +58,6 @@
     "express": "^4.14.1",
     "express-routebuilder": "^2.1.0",
     "express-session": "^1.15.1",
-    "fabric": "^1.7.6",
```

Comment:
This should have the added benefit of removing a dependency that requires building for each node version, a process that's been failing in our CI environment.